dialog/en-us/wait.for.startup.dialog

```diff
@@ -0,0 +1,2 @@
+pairing is now complete, wait one moment while skills are loading.
```

Comment:
consider changing to "wait a few moments".  depending on a users' internet connection, this could take several minutes